flycheck.el

```diff
@@ -5782,7 +5929,8 @@ See URL `http://clang.llvm.org/'."
               (or (flycheck-error-message err) "no message")))
       (flycheck-fold-include-levels errors "In file included from")))
   :modes (c-mode c++-mode)
-  :next-checkers ((warning . c/c++-cppcheck)))
+  :next-checkers ((warning . c/c++-cppcheck))
+  :conflicts-with c/c++-gcc)
```

Comment:
What would happen if we removed this `conflicts-with` clause? Both checkers would run, but would that be a problem?